app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_name/AddBusStopName.java

```diff
@@ -0,0 +1,50 @@
+package de.westnordost.streetcomplete.quests.bus_stop_name;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddBusStopName extends SimpleOverpassQuestType
+{
+
+	@Inject public AddBusStopName(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override protected String getTagFilters()
+	{
+		return " nodes, ways with !name and noname != yes" +
+			   " and ( highway = bus_stop or public_transport = platform )";
```

Comment:
The tactile paving quests filters that to only ask when it is not the stop position (i.e. `(highway=bus_stop and public_transport!=stop_position)`). So maybe you should do it here, too?